pymeasure/instruments/instrument.py

```diff
@@ -74,6 +77,12 @@ class Object(object):
         self.isShutdown = False
         log.info("Initializing %s." % self.name)
 
+    def __enter__(self):
```

Comment:
OK, let's put this in. Thank you for including a test! 
Please also add some explanation and short example to the documentation, so that this is actually discoverable. :-)

Reply:
Yes, will do!